.eslintrc

```diff
@@ -5,13 +5,10 @@ ecmaFeatures:
   - generators
 env:
   node: true
+  es6: true
```

Comment:
I'm currently fixing all of these in the new-entity-editing branch. I should've cleaned them up before I opened that. I'd like to leave it broken for now and fix it in the next PR, to avoid a ton of merge conflicts.